### DISCUSSION TITLE
milvus search on (5000,64) vectors with index type FLAT  very slow

### DISCUSSION THREAD

**Discussion by zhangjiulong** [milvus-io/milvus#23978] [Q&A and General discussion]
I search with milvus cost 0.38s per search
but with gensim similar_by_vector method cost only 3ms 
does anybody has some explantion?

**Comment by yhmo** [comment]
I guess you are using "Strong" consistency level to search. Try consistency_level="Bounded"

```
results = collection.search(
	data=vectors, 
	anns_field=vector_field_name, 
	param=search_params,
	limit=10, 
	consistency_level="Bounded"
)
```

**Comment by xiaofan-luan** [comment]
> I search with milvus cost 0.38s per search but with gensim similar_by_vector method cost only 3ms does anybody has some explantion?

Another suggestion is to not use FLAT Index, IVF_FLAT should be much faster
